.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@
     hooks:
     -   id: flake8
 -   repo: https://github.com/timothycrosley/isort
-    rev: ''
+    rev: '5.1.4'
```

Comment:
this is in direct conflict with https://github.com/yt-project/yt/pull/2777, I'll remove this after #2777 is merged